# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/uclibc/hotfix-h8300.patch.cross0
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- uClibc-ng-1.0.42/Rules.mak.vanilla	2023-01-03 17:45:13.491454316 +0100
+++ uClibc-ng-1.0.42/Rules.mak	2023-01-03 17:45:32.646453056 +0100
@@ -435,8 +435,8 @@
 endif
 
 ifeq ($(TARGET_ARCH),h8300)
-	CPU_CFLAGS-$(CONFIG_H8300H) += -mh -mint32
-	CPU_CFLAGS-$(CONFIG_H8S)    += -ms -mint32
+	#CPU_CFLAGS-$(CONFIG_H8300H) += -mh -mint32
+	#CPU_CFLAGS-$(CONFIG_H8S)    += -ms -mint32
 endif
 
 ifeq ($(TARGET_ARCH),cris)
